apps/site/src/app/about/getMembers.ts

```diff
@@ -1,28 +1,14 @@
-const FEED_URL = "https://docs.google.com/spreadsheets/d/";
-const SPREADSHEET_KEY = "1DWCOQBlzA3mpa2BXYXPmQrF9_-SpoPFbTdDQuCQ83hU";
-const QUERY = "pub";
-const FORMAT = "output=tsv";
-
-const dataURL = FEED_URL + SPREADSHEET_KEY + "/" + QUERY + "?" + FORMAT;
-
-const getSheetsData = async (page: string) => {
-	const response = await fetch(dataURL);
-
-	if (response.status !== 200)
-		console.warn(
-			"Error ocurred while fetching schedule data:",
-			response.statusText
-		);
-
-	const csv = await response.text();
-	for (const line of csv.split("\n")) {
-		const [key, value] = line.split("\t");
-		if (key === page) {
-			return JSON.parse(value);
-		}
-	}
-
-	return null;
-};
-
-export const getMembers = async () => await getSheetsData("members");
+import { cache } from "react";
+import { client } from "@/lib/sanity/sanityClient";
+
+export const getMembers = cache(async () => {
+	return await client
+		.fetch(
+			`*[_type == 'boardYear'][0]{corporate[]{person->{name, profilePic{asset->{url}}, socials[0]{link}}, position}, 
+                                  logistics[]{person->{name, profilePic{asset->{url}}, socials[0]{link}}, position}, 
+                                  marketing[]{person->{name, profilePic{asset->{url}}, socials[0]{link}}, position}, 
+                                  tech[]{person->{name, profilePic{asset->{url}}, socials[0]{link}}, position}}`
```

Comment:
This groq query can probably be condensed a bit.

Something like the following should work since we don't need the asset url anymore.

```groq
*[_type == 'boardYear'] | order(year desc) [0]{
  corporate[]{
    person->{
      name,
      profilePic,
      socials[0] {link}
    },
    position
  },
  logistics[]{
    person->{
      name,
      profilePic,
      socials[0] {link}
    },
    position
  },
  marketing[]{
    person->{
      name,
      profilePic,
      socials[0] {link}
    },
    position
  },
  tech[]{
    person->{
      name,
      profilePic,
      socials[0] {link}
    },
    position
  }, 
}
```

Reply:
Didn't catch this the first time around, but we probably shouldn't assume the linkedin link is the first on in the array. I'll come up with an update groq query later and reply here.